examples/comment/reply_to_media_comments.py

```diff
@@ -0,0 +1,74 @@
+# coding=utf-8
+"""
+    instabot example
+
+    Workflow:
+        If media is commented, reply to comments if you didn't reply yet to that user.
+"""
+
+import argparse
+import os
+import sys
+from tqdm import tqdm
+
+sys.path.append(os.path.join(sys.path[0], '../../'))
+from instabot import Bot
+
+parser = argparse.ArgumentParser(add_help=True)
+parser.add_argument('-u', type=str, help="username")
+parser.add_argument('-p', type=str, help="password")
+parser.add_argument('-proxy', type=str, help="proxy")
+parser.add_argument('-comments_file', type=str, help="comments_file")
+parser.add_argument('-link', type=str, help='media_link')
+args = parser.parse_args()
+
+if not args.comments_file:
+    print("You need to pass a path to the file with comments with option\n"
+          "-comments_file COMMENTS_FILE_NAME")
+    exit()
+if not args.link:
+    print("You need to pass the media link with option\n"
+          "-link MEDIA_LINK")
+    exit()
+
+if not os.path.exists(args.comments_file):
+    print("Can't find '{}' file.".format(args.comments_file))
+    exit()
+
+bot = Bot(comments_file=args.comments_file)
+bot.login(username=args.u, password=args.p,
+          proxy=args.proxy)
+
+media_id = bot.get_media_id_from_link(args.link)
+comments = bot.get_media_comments(media_id)
+if len(comments) == 0:
+    bot.logger.info("Media `{link}` has got no comments yet.".format(args.link))
+    exit()
+for comment in tqdm(comments):
+    replied = False
+    parent_comment_id = comment['pk']
+    user_id = comment['user']['pk']
+    comment_type = comment['type']
+    commenter = comment['user']['username']
+    text = comment['text']
+    # if using python3 change to:
+    # bot.logger.info("Checking comment `{text}` from `{commenter}`".format(text=text,commenter=commenter))
```

Comment:
Why we cannot use this line in python2?

Reply:
utf-8 errors (emoji etc), don't know how to fix for python2

Reply:
Solved with a try excpet

Reply:
Try to add at the top of module
```python
from __future__ import unicode_literals
```
Probably this will work without try-except.
If not - leave with try-except

Reply:
Nice! Good job!